ezbake/config/conf.d/auth.conf

```
@@ -56,6 +56,18 @@ authorization: {
             sort-order: 500
             name: "puppetlabs csr"
         },
+        {
+            # Allow nodes to renew their certificate
+            match-request: {
+                path: "/puppet-ca/v1/certificate_renewal"
+                type: path
+                method: post
+            }
+            # this endpoint should never be unauthenticated, as it requires the cert to be provided.
+            allow-unauthenticated: false
```

By default `allow-unauthenticated` is false, it turns out that we need to allow anything that provides a cert so: 

```suggestion
            # this endpoint should never be unauthenticated, as it requires the cert to be provided.
            "allow": "*",
```

---

See the "puppetlabs CA cert and CRL expiration" section below as an example of what I mean.